modules/support_ticket/support_ticket.module

```diff
@@ -266,6 +266,79 @@ function support_ticket_load($nid = NULL, $reset = FALSE) {
 }
 
 /**
+ * Determines what fields have changed when the support ticket is updated.
+ *
+ * @param object $support_ticket
+ *   The support ticket entity object.
+ */
+function support_ticket_entity_update($support_ticket) {
+  if (!method_exists($support_ticket, 'getFields')) {
+    return;
+  }
+
+  $fields = $support_ticket->getFields();
+  $langcodes = array_keys($support_ticket->getTranslationLanguages());
+  $original = !empty($support_ticket->original) ? $support_ticket->original: NULL;
+
+  // Don't log changes to the following fields when the support_ticket is updated.
+  // @todo: What's the cleanest way to expose this and make it configurable?
+  $ignore = array(
+    'field' => array('stid', 'uuid', 'vid', 'langcode', 'changed', 'revision_timestamp', 'revision_uid', 'revision_log', 'revision_translation_affected'),
+    'type' => array('comment'),
+  );
+  $items = array();
+  foreach ($fields as $name => $field) {
+    $type = $field->getFieldDefinition()->getType();
+    if (!in_array($name, $ignore['field']) && !in_array($type, $ignore['type'])) {
+      foreach ($langcodes as $langcode) {
+        $compare = $support_ticket->getTranslation($langcode)->get($name)->filterEmptyItems();
+        $compare_original = $original->getTranslation($langcode)->get($name)->filterEmptyItems();
+        if (!$compare->equals($compare_original)) {
+          $markup = array();
+          $view_builder = \Drupal::entityManager()->getViewBuilder($support_ticket->getEntityTypeId());
+          // @todo: expose and/or set sane display_mode
+          $value_original = $view_builder->viewField($original->getTranslation($langcode)->get($name));
+          if ($rendered = drupal_render($value_original)) {
+            $markup[] = "<span class='metadata-change-original'>-$rendered</span>";
+          }
+          // @todo: expose and/or set sane display_mode
+          $value = $view_builder->viewfield($support_ticket->getTranslation($langcode)->get($name));
+          if ($rendered = drupal_render($value)) {
```

Comment:
Can skip the early render here I think? Just do array('#prefix' => '<span....', $value, '#suffix' => '</span>')